src/test/java/fi/otavanopisto/kuntaapi/test/ManagementMenuMocker.java

```diff
@@ -0,0 +1,133 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.containing;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.junit.Assert.fail;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.tomakehurst.wiremock.matching.StringValuePattern;
+
+import fi.metatavu.management.client.model.Menu;
+import fi.metatavu.management.client.model.Menuitem;
+
+@SuppressWarnings ("squid:S1166")
+public class ManagementMenuMocker extends AbstractBaseMocker {
+
+  private static final String PATH_TEMPLATE = "%s/%s";
+  private static final String RESOURCES_PATH = "/wp-json/kunta-api/menus";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1075)